cli/daemon/test.go

```diff
@@ -67,12 +67,14 @@ func (s *Server) Test(req *daemonpb.TestRequest, stream daemonpb.Daemon_TestServ
 			}
 		}()
 
+		testEnv := append([]string{"ENCORE_RUNTIME_LOG=error", "ENCORE_LOG=error"}, req.Environ...)
```

Comment:
Do we need to raise the application log level to error too? I think it would be sufficient to bump it for the runtime

Reply:
Changed to only set the runtime log level. It might be a bit spammy still if you log from your handlers, would be nice to only show logs on failure or something, but maybe that is configurable in vitest